src/PostgresEventStore.php

```diff
@@ -81,6 +81,9 @@ public function __construct(
 
         Assertion::min($loadBatchSize, 1);
 
+        $this->setAttribute(PDO::ATTR_STATEMENT_CLASS, ['Doctrine\DBAL\Driver\PDOStatement', []]);
```

Comment:
Inspired from Doctrine/DBAL.

Reply:
but Doctrine/DBAL is not a dependency, so why do you set it as statement class?

Reply:
Sorry that was a mistake. I'll get rid of that.